lib/kafo/kafo_configure.rb

```diff
@@ -58,6 +58,7 @@ def initialize(*args)
       scenario_manager.check_enable_scenario
       scenario_manager.check_disable_scenario
       setup_config(config_file)
+      Logger.setup
```

Comment:
This feels very wrong. The config can affect the logger so you really can't initialize a logger before you have a config.

If you're keeping this, you must update the documentation. Right now it clearly states you have no logger setup before `init`.